### PR TITLE
[ESSNTL-5336] - For code clarity, rename staleness files, classes, and variables

### PR DESCRIPTION
**Pull request by jpramos123**
# Overview

This PR is being created to address [ESSNTL-5336](https://issues.redhat.com/browse/ESSNTL-5336).

In this PR it is being addressed the following changes:

1. Creation of a new table called `staleness` and removing the old table `account_staleness_culling`
2. Renaming files, methods and classes to not refer to the word `Account`
3. Update tests
4. Update OpenAPI schema with new naming 
5. Change the API response type to be integer instead of string

## PR Checklist

- [ ] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Uses [type hinting](https://docs.python.org/3/library/typing.html), if convenient
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by fellipeh — APPROVED** [review]
Good job!!

File: api/staleness_query.py [hunk 1]
```diff
@@ -48,19 +46,17 @@ def get_account_staleness_db(rbac_filter=None, identity=None):
...
     org_id = get_current_identity().org_id
-    account = get_current_identity().account_number
-    acc_st = _build_acc_staleness_sys_default(org_id, account)
+    acc_st = _build_acc_staleness_sys_default(org_id)
     return acc_st
 
 
-def _build_acc_staleness_sys_default(org_id, account):
+def _build_acc_staleness_sys_default(org_id):
     from app import inventory_config
```
Comment (thearifismail) [inline api/staleness_query.py:54]: better to move the import statement to the top of the file like others

**Review by kruai — DISMISSED** [review]
LGTM, though I'll leave the naming changes up to @thearifismail for final approval since he's the creator of the Jira. Thanks for this!

**Review by thearifismail — COMMENTED** [review]
Except of one minor comment, LGTM

**Review by kruai — APPROVED** [review]

**Review by fellipeh — APPROVED** [review]

**Pull request merged**